*** tests/A-009-008-M-ok.out	2016-12-05 13:13:09.000000000 +0000
--- tests/A-009-008-M-ok.outhyp	2016-12-08 06:36:06.447648945 +0000
***************
*** 26,34 ****
  5 - Mostrar os identificadores presentes no programa
  6 - Mostrar os identificadores do programa sem inicialização explícita
  0 - Sair
! Escolha uma opção: 1
! 4
! 0
  Menu de Edição
  1 - Listar programa
  2 - Executar programa
--- 26,34 ----
  5 - Mostrar os identificadores presentes no programa
  6 - Mostrar os identificadores do programa sem inicialização explícita
  0 - Sair
! Escolha uma opção: pex.core.expression.literal.IntegerLiteral@3764951d
! pex.core.expression.literal.IntegerLiteral@4b1210ee
! pex.core.expression.literal.IntegerLiteral@4d7e1886
  Menu de Edição
  1 - Listar programa
  2 - Executar programa
