*** tests/A-009-002-M-ok.out	2016-12-05 13:13:09.000000000 +0000
--- tests/A-009-002-M-ok.outhyp	2016-12-08 06:36:06.235646770 +0000
***************
*** 24,30 ****
  5 - Mostrar os identificadores presentes no programa
  6 - Mostrar os identificadores do programa sem inicialização explícita
  0 - Sair
! Escolha uma opção: false
  Menu de Edição
  1 - Listar programa
  2 - Executar programa
--- 24,30 ----
  5 - Mostrar os identificadores presentes no programa
  6 - Mostrar os identificadores do programa sem inicialização explícita
  0 - Sair
! Escolha uma opção: true
  Menu de Edição
  1 - Listar programa
  2 - Executar programa
